README.md

```diff
@@ -43,7 +43,7 @@ The following changes were deployed into production.
 
 ### cis-deductions-api
 
-* Some error response status codes corrected from 403 to 400, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
+Some error response status codes corrected from 403 to 400, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
```

Comment:
I think these need the asterisks for consistency, seems to be how the other PRs are being done